security/advancedtls/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.14
 require (
 	github.com/google/go-cmp v0.5.1
 	google.golang.org/grpc v1.31.0
-	google.golang.org/grpc/examples v0.0.0-20201020200225-9519efffeb5d
+	google.golang.org/grpc/examples v0.0.0-20201112215255-90f1b3ee835b
```

Comment:
This shouldn't matter since it's a replace - did you find that it mattered?  Or does `go mod tidy` automatically update this?  (Same with above)

Reply:
When I do a `make build` or `make test` from the root directory, these go.mod files are automatically updated.